src/services/completions.ts

```diff
@@ -706,10 +710,10 @@ export function getCompletionsAtPosition(
             return response;
         case CompletionDataKind.JsDocTagName:
             // If the current position is a jsDoc tag name, only tag names should be provided for completion
-            return jsdocCompletionInfo(JsDoc.getJSDocTagNameCompletions());
+            return jsdocCompletionInfo(JsDoc.getJSDocTagNameCompletions().concat(getJSDocParameterCompletions(sourceFile, position, preferences, compilerOptions, /*tagNameOnly*/ true)));

```

Comment:
I like `[...x, ...y]` instead of `x.concat(y)` because the type of `concat` is so inconsistent/complex.